.dev/tests/cypress/helpers.js

```diff
@@ -510,13 +546,7 @@ export function hexToRGB( hex ) {
 	return 'rgb(' + +r + ', ' + +g + ', ' + +b + ')';
 }
 
-function getIframeDocument( containerClass ) {
-	return cy.get( containerClass + ' iframe' ).its( '0.contentDocument' ).should( 'exist' );
-}
-
-export function getIframeBody( containerClass ) {
-	return getIframeDocument( containerClass ).its( 'body' ).should( 'not.be.undefined' )
-		// wraps "body" DOM element to allow
-		// chaining more Cypress commands, like ".find(...)"
-		.then( cy.wrap );
+// Mailserver is usually setupped on CI
+export function MailServerSetupped() {
+	return Cypress.env( 'testURL' ) !== 'http://localhost:8889';
```

Comment:
Are we use this helper function name needs to include MailServer? I am not seeing the relation between the output and Mail Server. Its really just checking the testURL, and nothing else. 

Maybe something like `IsNotWPTestEnv` or something like that would make more sense?

Reply:
Renamed it to a more relevant name, good idea.